apm-server/README.md

```diff
@@ -8,13 +8,18 @@ The design and code is less mature than official GA features and is being
 provided as-is with no warranties. Alpha features are not subject to the support
 SLA of official GA features (see [supported configurations][] for more details).
 
+**Warning**: This branch is used for development, please use [7.9.1][] release
+for released version.
+
+
```

Comment:
```suggestion
**Warning**: This branch is used for development, please use [7.9.1][] release for released version.
```

I was thinking about keeping this message on a single line now, this way we can drop it more easily with `sed -i -e "/This branch is used for development, please use .* release for released version\./d" */README.md` when bumping again to release version (see https://github.com/elastic/infra/blob/master/docs/releases/helm-charts.md#bump-the-release-branch-from-snapshot-to-release-version)